packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -33,6 +33,14 @@ export interface FieldChangeRebaser<TChangeset> {
         changes: TaggedChange<TChangeset>[],
         composeChild: NodeChangeComposer,
         genId: IdAllocator,
+        crossFieldManager: CrossFieldManager,
+    ): TChangeset;
+
+    amendCompose(
```

Comment:
Needs docs to explain what this is meant to do and call out the mutability (or not) of `composeChange`.
Same for other amend methods.